course-definition.yml

```diff
@@ -676,4 +688,47 @@ stages:
     primary_extension_slug: "pub-sub"
     name: "Unsubscribe"
     difficulty: medium
-    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
\ No newline at end of file
+    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
+
+  # Geospatial Commands
+  - slug: "zt4"
+    primary_extension_slug: "geospatial"
+    name: "Respond to GEOADD"
+    difficulty: easy
+    marketing_md: In this stage, you'll add support for responding to the `GEOADD` command.
+
+  - slug: "ck3"
+    primary_extension_slug: "geospatial"
+    name: "Validate coordinates"
+    difficulty: easy
+    marketing_md: In this stage, you'll add support for validating the latitude and longitude provided in the `GEOADD` command.
+
+  - slug: "tn5"
+    primary_extension_slug: "geospatial"
+    name: "Store a location"
+    difficulty: easy
+    marketing_md: In this stage, you'll add support for storing a location in a sorted set.
+
+  - slug: "cr3"
+    primary_extension_slug: "geospatial"
+    name: "Calculate location score"
+    difficulty: easy
```

Comment:
Let's adjust these difficulties based on your best guess @UdeshyaDhungana - don't think this one would be `easy` for example